--- adagucserverEC/Makefile	Wed Oct 22 15:17:44 2014 +0200
+++ adagucserverEC/Makefile	Mon Jan 05 17:06:16 2015 +0100
@@ -4,7 +4,7 @@
 CCDFDATAMODEL = ../CCDFDataModel
 CCDFDATAMODELOBJ = $(CCDFDATAMODEL)/CCDFDataModel.a
 
-INCLUDEDIR= $(CPPFLAGS) -I $(HELPERCLASSES) -I $(CCDFDATAMODEL) -I/usr/include/libxml2/  -I/usr/include/udunits2/ -I/usr/include/freetype2/ -I/usr/include/postgresql/ -I/usr/include/cairo/ -I/usr/include/gdal/
+INCLUDEDIR= $(CPPFLAGS) -I $(HELPERCLASSES) -I $(CCDFDATAMODEL)
 
 
 
